app/api/services/post.py

```
@@ -1,22 +1,25 @@
 from app.core.image import upload_img_to_cloudinary
+from app.api.utils.deps import SessionDep
 
 async def create_new_post(
-    db: Session, 
+    db: SessionDep, 
     title: str, 
     content: str, 
     images: List[UploadFile],
-    user_id: str
+    user_id: str,
+    hashtag: List[str] = None
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)